gateway/1.4/modules/ROOT/pages/_partials/task-node-affinity.adoc

```diff
@@ -0,0 +1,238 @@
+//TEMPORARY TAG for test builds
+//tag::temp-tag1[]
+= Select a Node for Flex Gateway Deployments
+
+Flex Gateway is often critical to the security of a cluster, so it is important to host it on a specialized node that meets specific requirements, instead of using the nodes that host your other deployments.
+
+To enable the Kubernetes scheduler to select a suitable host node for a Pod that contains Flex Gateway, you can incorporate custom node affinity rules from your Pod specification (Pod spec) into a Helm chart for your Flex Gateway Ingress Controller. For more information about node affinity, see the https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity[Kubernetes documentation^].
+
+[[list_add_labels]]
+== List and Create Node Labels
+
```

Comment:
Do we need to provide guidance on what sort of node labels we might want to create or what labels our flex pod spec might want to use in the node affinity rules? Are there any common node characteristics we'd want for our flex pod?

Reply:
not really. Users can have any label they want to identify their nodes